validate.js

```diff
@@ -77,28 +78,32 @@ export default class Validate extends Validates {
   }
 
   /**
-   * React lifecycle handler called immediately before the component's initial render.
+   * React lifecycle handler called immediately after the component's initial render.
```

Comment:
Why change this?

Reply:
`componentDidMount` is called after render, whereas `componentWillMount` is called before render, so I updated the JSDoc to reflect that. I made this change since `componentWillMount` is being deprecated.

Reply:
That makes sense. I forgot that `componentWillMount` was one of the lifecycle methods being deprecated.